word_counter.py

```diff
@@ -58,16 +58,19 @@ def show_total(self) -> int:
         return total
 
     @staticmethod
-    def get_files_by_extension(location: str, extension: str, preview=False, preview_size=395) -> int:
+    def get_files_by_extension(location: str, extension: str, preview=False, preview_size=395, recursion=False) -> int:
```

Comment:
here maybe -> recursion = True?

Reply:
Yes. Following the recursion pattern implemented elsewhere is a good idea. By default, it will use recursion, but the user may choose `--no-recursion`. So, it should default to `True`